lldb/packages/Python/lldbsuite/test/builders/darwin.py

```diff
@@ -97,6 +97,20 @@ def getArchCFlags(self, arch):
 
         return "ARCH_CFLAGS=\"-target {} {}\"".format(triple, version_min)
 
+    def getSwiftTargetFlags(self, arch):
+        if not arch:
+            arch = configuration.arch
+        if not arch:
+            return ""
+        vendor, os, version, env = get_triple()
+        if not vendor or not os or not version or not env:
+            return ""
+        flags = 'TARGET_SWIFTFLAGS="-target {}-{}-{}{}"'.format(
+            arch, vendor, os, version)
+        if env:
+            flags+="-"+env
```

Comment:
When there's `env` value, the quoting will be off. For example if `env` is `"simulator"`, the value of `flags` will be:

```
TARGET_SWIFTFLAGS="-target {}-{}-{}{}"-simulator
```

A shell will accept that, but it's probably good to get the `env` inside the quotes.

Reply:
Good point!